qutebrowser/browser/downloads.py

```diff
@@ -131,7 +131,11 @@ def create_full_filename(basename, filename):
     encoding = sys.getfilesystemencoding()
     filename = utils.force_encoding(filename, encoding)
     basename = utils.force_encoding(basename, encoding)
-    if os.path.isabs(filename) and os.path.isdir(filename):
+
+    if filename.endswith('/'):
```

Comment:
FWIW things like mutt use the default filename under that directory then, which seems a sensible thing to do to me.